meta2v2/meta2_filters_action_content.c

```diff
@@ -98,10 +98,15 @@ _m2b_notify_beans(struct meta2_backend_s *m2b, struct oio_url_s *url,
 	if (!send_chunks) {
 		GSList *non_chunks = NULL;
 		for (GSList *l = beans; l; l = l->next) {
-			if (DESCR(l->data) != &descr_struct_CHUNKS)
+			if (DESCR(l->data) == &descr_struct_ALIASES) {
+				if (alias)
+					GRID_WARN("Several aliases in same event!");
```

Comment:
It would be a good idea to give more information: which alias? which content id? which request id?